client/src/components/PeerReviewDetails/PeerReviewDetails.js

```diff
@@ -47,18 +47,18 @@ const PeerReviewDetails = props => {
       [
         <span className={classes.boldFont}>Employee Under Review</span>,
         reviewData.employee_under_review.firstname +
-          ' ' +
-          reviewData.employee_under_review.lastname,
+        ' ' +
+        reviewData.employee_under_review.lastname,
```

Comment:
I think we need to use template literal here. Is it possible to use template literal?

Reply:
Did in PR #31